src/webview/html/messageAsHtml.js

```diff
@@ -92,7 +92,18 @@ const briefMessageAsHtml = ({
 }: BriefMessageProps) => template`
 $!${messageDiv(id, 'message-brief', flags)}
   <div class="content">
-    $!${messageBody({ content, flags, id, isOutbox, ownEmail, reactions, realmEmoji, timeEdited })}
+    <div dir="auto" class="message-content">
```

Comment:
You don't need to add the class as you have not added any style to it.
Here you can also add the `dir="auto"` directly to the `div` above.